### PR TITLE
Feature/task234

### PR DESCRIPTION
**Pull request by andrevargas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # generalTests
-For internal purposes only
+For internal purposes only 
+1 - FIRST COMMIT
+2 - Second commit
```
Comment (HuenderBr) [inline README.md]: Erro

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # generalTests
```
Comment (HuenderBr) [inline README.md]: Altera o hue

File: README.md [hunk 3]
```diff
@@ -1,2 +1,4 @@
...
-For internal purposes only
+For internal purposes only 
+1 - FIRST COMMIT
```
Comment (HuenderBr) [inline README.md]: Teste

File: README.md (hunk 1 above)
Comment (andrevargas) [reply README.md]: * [ ] hey
* [ ] ho

**Pull request merged**